kubernetes/opentelemetry-demo.yaml

```diff
@@ -339,7702 +207,139 @@ data:
           - batch
           receivers:
           - otlp
+          - jaeger
+          - zipkin
```

Comment:
do we need these or should they be disabled in the helm file?

Reply:
These are receivers, not exporters. I assumed they were being used to ingest traces (that are then sent to googlecloud).